[2024-06-19T08:42:08.070+0545] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T08:42:08.127+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-19T02:57:06.500432+00:00 [queued]>
[2024-06-19T08:42:08.146+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-19T02:57:06.500432+00:00 [queued]>
[2024-06-19T08:42:08.146+0545] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T08:42:08.254+0545] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 02:57:06.500432+00:00
[2024-06-19T08:42:08.256+0545] {standard_task_runner.py:63} INFO - Started process 15821 to run task
[2024-06-19T08:42:08.260+0545] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'manual__2024-06-19T02:57:06.500432+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/basic_dag.py', '--cfg-path', '/tmp/tmprywf111q']
[2024-06-19T08:42:08.263+0545] {standard_task_runner.py:91} INFO - Job 205: Subtask backup_task
[2024-06-19T08:42:08.450+0545] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task manual__2024-06-19T02:57:06.500432+00:00 [running]> on host manisha-Inspiron-3580
[2024-06-19T08:42:08.807+0545] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='manisha' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T02:57:06.500432+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-19T02:57:06.500432+00:00'
[2024-06-19T08:42:08.809+0545] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T08:42:08.841+0545] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T08:42:08.842+0545] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/manisha/airflow/bashScripts/backup.sh ']
[2024-06-19T08:42:08.860+0545] {subprocess.py:86} INFO - Output:
[2024-06-19T08:42:08.876+0545] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T08:42:08.907+0545] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T08:42:08.918+0545] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T08:42:09.318+0545] {subprocess.py:93} INFO - Backup completed in 1 seconds
[2024-06-19T08:42:09.318+0545] {subprocess.py:93} INFO - Backup is Successfully done in location: /home/manisha/airflow/backup|Required Space: 0GB, Available Space: 428.21GB
[2024-06-19T08:42:09.318+0545] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T08:42:09.319+0545] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T08:42:09.423+0545] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=manual__2024-06-19T02:57:06.500432+00:00, execution_date=20240619T025706, start_date=20240619T025708, end_date=20240619T025709
[2024-06-19T08:42:09.638+0545] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T08:42:09.742+0545] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T08:42:09.794+0545] {local_task_job_runner.py:222} INFO - ::endgroup::
